src/radical/pilot/task_description.py

```diff
@@ -705,6 +712,9 @@ def _verify(self):
             self.raptor_class = self.worker_class
             self.raptor_class = ''
 
+        if self.use_mpi is None:
+            self.use_mpi = bool(self.ranks)
```

Comment:
`True if self.ranks > 1 else False` (`bool(self.ranks)` will always return `True` in our case)